### PR TITLE
qt: Splashscreen redesign

### PR DESCRIPTION
**Pull request by xdustinface**
This PR ist part of a series of PRs related to UI redesigns. Its ancestor is #3575.
 
I hope that no one has a good reason to keep the copyrights on the splashscreen because i removed them in this PR. I thought it's enough to have them presented in the "About Dialog". If you see any problem with it let me know but i felt like i need to try it because it just looks much cleaner :)  

#### Changes of this PR 
- Makes it theme related dark or light
- Fixes the blurriness issues
- Fixes issues with overlapping init messages at the bottom of the
splashscreen
- Give it a cleaner look -> Removed copyrights and prefix `Version v` of the version string

Old Testnet             |  Old Mainnet
:-------------------------:|:-------------------------:
 | 

Light New Testnet             |  Light New Mainnet
:-------------------------:|:-------------------------:
 | 

Dark New Testnet             |  Dark New Mainnet
:-------------------------:|:-------------------------:
 |

### CODE REVIEW

**Comment by UdjinM6** [comment]
I'm kind of on the fence with this one... On one hand, I like it, look much cleaner and reacts to theme changes but on the other hand, applying different colors to the logo kind of breaks the style guide I think (should be either Dash's blue or white, no other colors iirc).

**Comment by xdustinface** [comment]
Hmm, i showed suba both new splashscreens and he approved them (orange one of testnet shouldn't matter for styleguide consistency imo). But i will ask him again what he thinks especially about coloring the icons this way!

**Comment by xdustinface** [comment]
I talked to suba and we got the approval for this PR. The style guide will become adjusted in order to allow dark/light mode compatible blue dash logos and for testing usage there will be a new option to use the orange logo as we have it in here. Though, we still have regtest mode where we currently have a red logo with this PR. To minimize coloring of the logo and because its probably just not that important to have a seperate color for regtest we agreed on just using the same orange for all network modes other than mainnet, aka 
- mainnet => theme related blue
- testnet, devnet, regtest => orange 

@UdjinM6 thoughts?

**Comment by UdjinM6** [comment]
I'm not sure. Using a slightly different blue for the dark theme is fine IMO. But after playing with the orange one a bit I think using colors other than a blue-ish one (or white color like we currently do) for the logo is a bit too much. Also, I kind of like to have different tray icon colors for different networks. How about tweaking it like that https://github.com/UdjinM6/dash/commits/pr3613 instead? (the way I get these colors is a bit hack-ish but it does the job :) )

Light Regtest             |  Dark Testnet
:-------------------------:|:-------------------------:
Screenshot 2020-07-25 at 04 43 44 | Screenshot 2020-07-25 at 04 44 41

**Comment by xdustinface** [comment]
> I'm not sure. Using a slightly different blue for the dark theme is fine IMO. But after playing with the orange one a bit I think using colors other than a blue-ish one (or white color like we currently do) for the logo is a bit too much. 

Hm yeah right.. actually its not even really needed to use an orange logo. Your alternative solution is good enough to highlight the other networks, picked it 👍 

> (the way I get these colors is a bit hack-ish but it does the job :) )

it is haha

**Comment by UdjinM6** [comment]
> it is haha

😄

Let's fix this, pls see e95b21c8d47a1b07ae9149b5cef6b628d9b8792f and 50f4813c35a7a88eb4c462e46d6c6bda752e0c7a

**Comment by xdustinface** [comment]
Somehow it still doesn't feel _final_ but i can't think about the ultimate solution right now.. i guess i would prefer to remove the color shift and just apply direct coloring to the icons but thats has also some drawbacks if icons change lol.. Anyway, your fix is still much better than the hack-ish one before, picked! 😄

File: src/qt/splashscreen.cpp [hunk 1]
```diff
@@ -37,70 +38,94 @@ SplashScreen::SplashScreen(Qt::WindowFlags f, const NetworkStyle *networkStyle)
...
+    QFont fontBold = GUIUtil::getFontBold();
+
+    QPixmap pixmapLogo = networkStyle->getSplashImage();
+    pixmapLogo.setDevicePixelRatio(scale);
+
+    // Adjust logo color based on the current theme
+    QImage imgLogo = pixmapLogo.toImage().convertToFormat(QImage::Format_ARGB32);
+    QColor logoColor = GUIUtil::getThemedQColor(GUIUtil::ThemedColor::BLUE);
+    for (int x = imgLogo.width(); x--;) {
+        for (int y = imgLogo.height(); y--;) {
```
Comment (PastaPastaPasta) [inline src/qt/splashscreen.cpp:69]: Hmmm, I don't love these for loops.

Can we at do:
```
for (int x = imgLogo.width(); x > 0;) {
    x--;
...
```

Or at least
`for (int x = imgLogo.width(); x-- > 0;) {` ?

Comment (UdjinM6) [reply src/qt/splashscreen.cpp:69]: Hmmm. IMO these loops are ok-ish but if we are going to change them I'd prefer a more classical form i.e. `for (int x = imgLogo.width(); x > 0; --x) {`

**Pull request merged**